operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-accessor-serviceaccount.yaml

```diff
@@ -2,6 +2,5 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: observability-alertmanager-accessor
-  namespace: open-cluster-management
```

Comment:
Is this expected? In which namespace will this land?

Reply:
Service account resources are handled by `"ServiceAccount":        r.renderer.RenderNamespace,`
So it will land as before in the `namespace := mcoconfig.GetDefaultNamespace()` which is `open-cluster-management-observability`
I removed all of them so that there is no ambiguity. And they are all checked in unit tests.